src/youtube-music.css

```diff
@@ -96,3 +96,8 @@ tp-yt-iron-dropdown,
 tp-yt-paper-dialog {
   app-region: no-drag;
 }
+
+/* Fix duplicated scrollbar */
+body {
+  overflow: hidden !important;
```

Comment:
I'd prefer `overflow-y` since this is for the Y axis, but if someone gets a horizontal scrollbar then ngl that's their problem :rofl: